docs/django.rst

```diff
@@ -22,7 +22,7 @@ as a reference. What we need are basically these steps:
     def index(request):
         return HttpResponse("Hello, world. Everyone can read this line.")
 
-Identity-for-Django configuration
```

Comment:
FYI: At one point I added those explicit "... for Django" wording because the docs for different frameworks look too similar with each other. I worried the audience might accidentally land on an incorrect page (say, they meant to develop a Flask app but was reading a Django page). That being said, we can try your simplification and see how that goes.

Reply:
Ah okay, was wondering about that. Hopefully SEO gets them to the right place!